### PR TITLE
Add selecting and waiting PR widget for avoiding unstable the tests for factory package

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* **We have the next problem in current 'factory' tests:**
after consuming a factory and cloning a project, the project is opened for further checks. The method for opening at first step selects a project and next sends double click. But after selecting - may be opened PR widget. In this case - focus switches to the PR widget and double click does not work properly.
* **Solution:** in the depended tests - at first step performs selection and then - waiting for of opening PR. After it - the remaining steps.
- Also have been removed unnecessary try/catch blocks with closed issue.
- Add new try/catch block into `DirectUrlFactoryWithSpecificBranch` test with new issue

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/8458
@dmytro-ndp, @vparfonov, @Ohrimenko1988

### CODE REVIEW

**Review by Ohrimenko1988 — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java [hunk 1]
```diff
@@ -84,14 +87,14 @@ public void factoryWithDirectUrlWithSpecificBranch() throws Exception {
...
+    projectExplorer.selectItem("gitPullTest");
+    pullRequestPanel.waitOpenPanel();
 
     try {
       projectExplorer.expandPathInProjectExplorer("gitPullTest/my-lib");
-    } catch (TimeoutException ex) {
-      // remove try-catch block after issue has been resolved
-      fail("Known issue https://github.com/eclipse/che/issues/8458");
+    } catch (WebDriverException ex) {
+      fail("Known issue https://github.com/eclipse/che/issues/8616");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranch.java]: missed comment line

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java [hunk 2]
```diff
@@ -146,15 +148,11 @@ private void createFactoryAndSwitchToWs() throws IOException {
...
-          PROJECT_NAME + "/" + KEEPED_DIR + "/src" + "/main" + "/java/hello",
-          "GreetingController.java");
-    } catch (TimeoutException ex) {
-      // remove try-catch block after issue has been resolved
-      fail("Known issue https://github.com/eclipse/che/issues/8458");
-    }
+    projectExplorer.selectItem(PROJECT_NAME);
+    pullRequestPanel.waitOpenPanel();
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        PROJECT_NAME + "/" + KEEPED_DIR + "/src" + "/main" + "/java/hello",
```
Comment (vparfonov) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java]: :)

**Review by dmytro-ndp — APPROVED** [review]

**Comment by musienko-maxim** [comment]
ci-build

**Review by vparfonov — APPROVED** [review]

**Comment by musienko-maxim** [comment]
ci-build

**Pull request merged**